portfolio-alex.css

```diff
@@ -0,0 +1,94 @@
+*{
+	box-sizing:border-box;
+}
+
+body{
+	margin:0;
+	padding:0;
+}
```

Comment:
To prepare for making your code accessible/responsive, you can add the body font-size to 62.5%, and then change font-sizes within your code to rems. 1rem is approx 16px.